### PR TITLE
Subject and category in metadata fixed

### PR DESCRIPTION
**Pull request by Aca-jov**
SDNTB-208

### CODE REVIEW

File: scripts/superdesk-archive/views/metadata-view.html [hunk 1]
```diff
@@ -47,11 +47,15 @@
...
         <dl>
             <dt translate>ANPA Category</dt>
-            <dd ng-if="item['anpa_category']">{{ item['anpa_category'] | mergeWords:'name' }}</dd>
+            <dd ng-if="item.anpa_category">{{ item.anpa_category | mergeWords:'name' }}</dd>
         </dl>
-        <dl >
+        <dl>
             <dt translate>Subject</dt>
-            <dd ng-if="item.subject && item.subject.length > 0">{{ item.subject | mergeWords:'name'}}</dd>
+            <dd>{{ item.subject | mergeWords:'name':'subject_custom'}}</dd>
```
Comment (ioanpocol) [inline scripts/superdesk-archive/views/metadata-view.html]: This subject_custom is just a cv name, so it should not be hardcoded here.

File: scripts/superdesk-archive/views/metadata-view.html [hunk 2]
```diff
@@ -47,11 +47,15 @@
...
+        </dl>
+        <dl>
+            <dt translate>Category</dt>
```
Comment (ioanpocol) [inline scripts/superdesk-archive/views/metadata-view.html]: Category is a custom cv for NTB, so it should not appear here. The solution is to read the list of cv for current item's profile and show them dynamically in the way that are shown on item header.

**Comment by petrjasek** [comment]
:+1:

**Comment by ioanpocol** [comment]
:+1:

**Pull request merged**